app/src/main/java/de/danoeh/antennapod/fragment/SubscriptionFragment.java

```diff
@@ -278,6 +295,18 @@ public void update(EventDistributor eventDistributor, Integer arg) {
         }
     };
 
+    @Subscribe(sticky = true, threadMode = ThreadMode.MAIN)
+    public void onEventMainThread(DownloadEvent event) {
+        Log.d(TAG, "onEventMainThread() called with: " + "event = [" + event + "]");
+        DownloaderUpdate update = event.update;
+        if (isUpdatingFeeds != update.feedIds.length > 0) {
```

Comment:
Refactoring suggestion: centralize the update check condition to be a more descriptive method shared across fragments, e.g.,

```java
    if (event.hasFeedsUpdateStatusChanged(isUpdatingFeeds)) {
```